packages/medusa/src/api/routes/admin/gift-cards/list-gift-cards.ts

```diff
@@ -39,7 +39,7 @@ export default async (req, res) => {
     select: defaultAdminGiftCardFields,
     relations: defaultAdminGiftCardRelations,
     order: { created_at: "DESC" },
-    limit: validated.limit,
+    take: validated.limit,
```

Comment:
suggestion (non-blocking): I would suggest that during the migration if we touch end point that can be changed to use the transform middlewares we should migrate them as well. Wdyt?